github_wrapper/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+requests = "==2.20.0"
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
Please remove this requires section. So that your project is usable with Python 2.7 and earlier Python 3 versions. 
Python 2 is actively used and we need to support it as well. Write code that works for both

Reply:
I didn't add this line specifically. It got added when I installed pipenv later.  Removing.